translator/totomlconfig/sampleConfig/complete_linux_config.json

```diff
@@ -4,28 +4,38 @@
     "logfile": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
     "internal": true,
     "debug": true,
+    "aws_sdk_log_level": "LogDebug",
     "user_agent": "CUSTOM USER AGENT VALUE",
     "credentials": {
       "role_arn": "global_role_arn_value"
     }
   },
   "metrics": {
     "metrics_collected": {
-      "collectd":{
+      "collectd": {
         "service_address": "udp://127.0.0.1:25826",
         "name_prefix": "collectd_",
         "collectd_auth_file": "/etc/collectd/auth_file",
         "collectd_security_level": "encrypt",
-        "collectd_typesdb": ["/usr/share/collectd/types.db"],
+        "collectd_typesdb": [
+          "/usr/share/collectd/types.db"
+        ],
         "metrics_aggregation_interval": 60
       },
       "cpu": {
         "resources": [
           "*"
         ],
         "measurement": [
-          {"name": "cpu_usage_idle", "rename": "CPU_USAGE_IDLE", "unit": "unit"},
-          {"name": "cpu_usage_nice", "unit": "unit"},
+          {
+            "name": "cpu_usage_idle",
+            "rename": "CPU_USAGE_IDLE",
+            "unit": "unit"
+          },
+          {
+            "name": "cpu_usage_nice",
+            "unit": "unit"
+          },
```

Comment:
Oops, my IDE must have formatted JSON stuff.

Reply:
I think we could potentially introduce a format file so that we can ignore sample config files for formatting. I'm not really bothered by the format diff.